### PR TITLE
[cxxmodules] Enable 31 tests and exclude 6 tests

### PR DESCRIPTION
**Pull request by yamaguchi1024**
Recent change to master fixed some tests which were excluded previously, so I reverted "excluding tests" patches and excluded six tests which were failing in current master.

### CODE REVIEW

**Comment by yamaguchi1024** [comment]
@phsft-bot build just on slc6/gcc62 with flags -Druntime_cxxmodules=On -Dctest_test_exclude_none=on

**Comment by yamaguchi1024** [comment]
@phsft-bot build

**Comment by yamaguchi1024** [comment]
@phsft-bot build just on slc6/gcc62 with flags -Druntime_cxxmodules=On -Dctest_test_exclude_none=on

**Comment by yamaguchi1024** [comment]
@phsft-bot build just on slc6/gcc62 with flags -Druntime_cxxmodules=On -Dctest_test_exclude_none=on

**Comment by yamaguchi1024** [comment]
@phsft-bot build just on slc6/gcc62 with flags -Druntime_cxxmodules=On -Dctest_test_exclude_none=on

**Comment by yamaguchi1024** [comment]
@phsft-bot build just on slc6/gcc62 with flags -Druntime_cxxmodules=On -Dctest_test_exclude_none=on

File: cling/dict/fwd-decl-stdless/CMakeLists.txt [hunk 1]
```diff
@@ -1,8 +1,6 @@
-# FIXME: Temporary workaround for runtime_cxxmodule
-if(NOT ROOT_runtime_cxxmodules_FOUND)
 ROOTTEST_GENERATE_DICTIONARY(lessyDict       lessy.h       LINKDEF lessyLinkDef.h)
+
```
Comment (vgvassilev) [inline cling/dict/fwd-decl-stdless/CMakeLists.txt]: Can you remove this new line?

File: root/math/genvector/CMakeLists.txt [hunk 2]
```diff
@@ -7,8 +7,6 @@ ROOTTEST_ADD_TEST(coordinates4D
 ROOTTEST_ADD_TEST(rotationApplication
                   MACRO ${ROOT_SOURCE_DIR}/math/genvector/test/rotationApplication.cxx+)
 
-# FIXME: Temporary workaround for runtime_cxxmodules;
-if(NOT ROOT_runtime_cxxmodules_FOUND)
 ROOTTEST_ADD_TEST(testGenVector
                   MACRO ${ROOT_SOURCE_DIR}/math/genvector/test/testGenVector.cxx+)
-endif()
+
```
Comment (vgvassilev) [inline root/math/genvector/CMakeLists.txt]: Likewise.

File: root/meta/CMakeLists.txt [hunk 3]
```diff
@@ -51,15 +54,13 @@ ROOTTEST_ADD_AUTOMACROS(DEPENDS ANSTmpltInt.C TmpltInt0.C TmpltInt1.C TmpltFloat
                                 TmpltNoSpec.C Event.cxx ${COMPILE_MACRO_TEST})
 endif()
 
-# FIXME: Temporary workaround for runtime_cxxmodules;
-if(NOT ROOT_runtime_cxxmodules_FOUND)
+
```
Comment (vgvassilev) [inline root/meta/CMakeLists.txt]: Likewise.

**Comment by yamaguchi1024** [comment]
@vgvassilev These newlines are from reverts, do you want to remove them?

**Comment by vgvassilev** [comment]
@yamaguchi1024, ok, then ignore and go ahead.

**Pull request merged**